v22.1/show-jobs.md

```diff
@@ -135,15 +144,16 @@ You can filter jobs by using `SHOW AUTOMATIC JOBS` as the data source for a [`SE
 ~~~
 
 ~~~
-    job_id           |       job_type      |                    description                      | ...
-+--------------------+---------------------+-----------------------------------------------------+ ...
-  438235476849557505 | AUTO CREATE STATS   | Table statistics refresh for defaultdb.public.users | ...
-(1 row)
+        job_id       |         job_type          |                             description                             |                                         statement                                          | user_name |  status   | ...
+  786483120403382274 | AUTO SQL STATS COMPACTION | automatic SQL Stats compaction                                      |                                                                                            | node      | succeeded | ...
+  786476180299579393 | AUTO CREATE STATS         | Table statistics refresh for movr.public.promo_codes                | CREATE STATISTICS __auto__ FROM [110] WITH OPTIONS THROTTLING 0.9 AS OF SYSTEM TIME '-30s' | root      | succeeded | ...
+...
+(7 rows)
 ~~~
 
 ### Show changefeed jobs
 
- You can display specific fields relating to changefeed jobs by running `SHOW CHANGEFEED JOBS`. These fields include:
+You can display specific fields relating to changefeed jobs by running `SHOW CHANGEFEED JOBS`. These fields include:
 
 * [`high_water_timestamp`](monitor-and-debug-changefeeds.html#monitor-a-changefeed): Guarantees all changes before or at this time have been emitted.
```

Comment:
Not part of your changes, but we use `-` rather than `*` in lists and it wouldn't hurt to change these opportunistically.